packages/hdwallet-gridplus/src/gridplus.ts

```
@@ -0,0 +1,673 @@
+  }
+
+  public async getLabel(): Promise<string> {
+    return "GridPlus Lattice1";
+  }
+
+  public async getFirmwareVersion(): Promise<string> {
+    return "1.0.0";
+  }
+
+  public async getDeviceID(): Promise<string> {
+    return this.activeWalletId || await this.transport.getDeviceID();
```

ESLint

[error] Replace `await·this.transport.getDeviceID(` with `(await·this.transport.getDeviceID()`

(prettier/prettier)